src/pages/publication/references_tab/PublicationChannelSearch.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+
+import { getPublicationChannels } from '../../../api/external/publicationChannelApi';
+import { AutoSearch } from '../../../components/AutoSearch';
+import { searchFailure } from '../../../redux/actions/searchActions';
+import { PublicationChannel } from '../../../types/references.types';
+import useDebounce from '../../../utils/hooks/useDebounce';
+
+interface PublicationChannelSearchProps {
+  setFieldValue: (name: string, value: any) => void;
+}
+
+const PublicationChannelSearch: React.FC<PublicationChannelSearchProps> = ({ setFieldValue }) => {
+  const [searchResults, setSearchResults] = useState<PublicationChannel[]>([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searching, setSearching] = useState(false);
```

Comment:
hva brukes egentlig searching til? kan ikke se at vi mister noe ved å bare fjerne det..?

Reply:
Er vel for å ungå å søke flere ganger når man skriver inn tekst